### PR TITLE
ci: disable test for python 3.6

### PR DESCRIPTION
**Pull request by naveen521kk**
## List of Changes
I think it is time to remove python 3.6 from testing and soon remove support for it. This is done because running CI takes longer than expected and testing three major python version seem satisfying. This is not depreciating it but just not running the tests in CI.
- Don't test python 3.6

## Motivation
CI takes a lot of time to run and has pending.

## Explanation for Changes
Remove testing for python 3.6.

## Acknowledgements
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have added an entry describing the changes from this PR to the [Changelog](https://docs.manim.community/en/latest/changelog.html) at `docs/source/changelog.rst`

### CODE REVIEW

**Comment by leotrs** [comment]
Pro of this PR: the macos py3.6 test seems to consistently take the longest time to run.

Con of this PR: python3.6 reaches end of life at the end of next year. 

Perhaps we can disable py3.6 tests for now but still try to provide help/support for users on py3.6 if they run into issues. 

Although it might just be better to forget about 3.6 altogether already...

**Comment by naveen521kk** [comment]
> Perhaps we can disable py3.6 tests for now but still try to provide help/support for users on py3.6 if they run into issues.

I think we can do this.

**Comment by leotrs** [comment]
cc @eulertour

**Comment by kolibril13** [comment]
Is it possible to remove this test in the pr workflow and shift it to another GitHub action that runs it once every week?

**Comment by naveen521kk** [comment]
I don't think it is required. Simply disable test for python 3.6 but not actually drop support for it.

**Review by kolibril13 — APPROVED** [review]

**Review by leotrs — APPROVED** [review]

**Comment by naveen521kk** [comment]
Any objection in merging this?

**Comment by leotrs** [comment]
Not from me ,just waiting on pending review by @eulertour

**Review by eulertour — APPROVED** [review]

**Pull request merged**
